internal/api/api.go

```diff
@@ -308,35 +313,15 @@ func (s *statusResponse) SetState(state model.PodState) *statusResponse {
 	return s
 }
 
-func (api *idler) Status(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	response := statusResponse{}
-
-	openshiftURL, openshiftToken, err := api.getURLAndToken(r)
-	if err != nil {
-		response.AppendError(tokenFetchFailed, "failed to obtain openshift token: "+err.Error())
-		writeResponse(w, http.StatusBadRequest, response)
-		return
-	}
+type any interface{}
 
-	state, err := api.openShiftClient.State(
-		openshiftURL, openshiftToken,
-		ps.ByName("namespace"),
-		"jenkins",
-	)
+func writeResponse(w http.ResponseWriter, status int, response any) {
+	err := json.NewEncoder(w).Encode(response)
 	if err != nil {
-		response.AppendError(openShiftClientError, "openshift client error: "+err.Error())
-		writeResponse(w, http.StatusInternalServerError, response)
+		respondWithError(w, http.StatusInternalServerError, fmt.Errorf("Could not serialize users: %s", err))
```

Comment:
is `users` a copy-paste error?

Reply:
everything is a copy-paste error :laughing: